### DISCUSSION TITLE
Error: "grep must be installed on the remote system" in IFS browser

### DISCUSSION THREAD

**Discussion by stevabo** [codefori/vscode-ibmi#1831] [Q&A] [answered]
When I want to "search directory" in IFS Browser, I get the message "grep must be installed on the remote system for the IFS search". 
Anyone knows how to fix this?

Steven

**Comment by sebjulliand** [comment, marked as answer]
Hi @stevabo ,
you need the `grep` command line tool to be installed on your IBM i.

To do so, open a terminal or connect through SSH and run the following command: `yum install grep-gnu`.

Hope this helps 🙂

**Reply by stevabo** [reply]
Hi Sébastien,

Thanks for the quick response!

I see that the grep-gnu package is installed (with ACS open source package management). Did a reinstall but still not working.

**Reply by worksofliam** [reply]
@stevabo before you connect to your server, right click on it and select 'connect and reload settings' - then try again!

**Reply by stevabo** [reply]
Thanks @sebjulliand and @worksofliam, it works now.
